client/src/components/Stars/Stars.style.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+//Function that receives a multiple of 5 between 0-50 and returns a star component
+export const StarCreator = rating => {
+  let horizontal;
+  let vertical;
+  if (rating % 10 === 0) {
+    horizontal = -180 + rating * 3.6;
+    vertical = 0;
+  } else {
+    horizontal = -180 + (rating + 5) * 3.6;
+    vertical = -48;
+  }
+  const Stars = styled.span`
```

Comment:
I think Stars should be a helper that returns a styled component. I don't think it belongs in this function.

Reply:
maybe the vertical and height calculation should be a helper function, but it's fine if this file returns a styled component for consistency

Reply:
Guys- you're giving contradictory advice. What should we do?

Reply:
@jema28 - can you confirm that this is what you meant?

Reply:
@jema28 , @Karyum , I feel that the maths here seems simple enough to be included in this function.

Reply:
@jema28 , @Karyum . Has been modularised. Let me know what you think